data-analytics/bigquery-cross-project-slot-monitoring/tests/main_test.py

```diff
@@ -140,5 +140,4 @@ def testCopyMetrics_Valid(self, mock_copy_metrics):
       headers={'X-AppEngine-QueueName': 'SomeQueue'},
       status=200)
 
-    mock_copy_metrics.assert_called_once_with(
-    payload['src_project'], payload['dst_project'])
+    mock_copy_metrics.assert_called_once()
```

Comment:
Here's a good description of how to handle unit tests when your code is dependent on current time:

https://stackoverflow.com/questions/43799206/python-how-do-i-mock-datetime-utcnow

Please follow that pattern to bring your test strength back up (asserting on the method contents).

Reply:
Done. PTYAL.